apps/bc-wallet-showcase-creator/components/login-form.tsx

```diff
@@ -7,9 +7,24 @@ import { Button } from '@/components/ui/button'
 import { useTranslations } from 'next-intl'
 import { Link } from '@/i18n/routing'
 import { signIn } from 'next-auth/react'
+import { usePathname } from 'next/navigation'
 
 export function LoginForm({ className, ...props }: React.ComponentPropsWithoutRef<'div'>) {
   const t = useTranslations('login')
+  const pathname = usePathname()
+  const parts = pathname.split('/')
+  const locale = parts[1] || 'en'
+  const tenantId = parts[2]
+
+  const handleLogin = () => {
+
+    document.cookie = `tenantId=${tenantId}; path=/; max-age=3600;` // 1 hour
```

Comment:
can be much bigger, and when you press logout, you remove the cookie, probably the same use will not keep trying to login to multiple tenants in a day